Description: Disable http3 feature and related deps, deemed unstable upstream
Forwarded: not-needed
Last-Update: 2024-11-02
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -71,7 +71,7 @@
     "hyper-rustls?/ring",
     "tokio-rustls?/ring",
     "rustls?/ring",
-    "quinn?/ring",
+#    "quinn?/ring",
 ]
 __tls = [
     "dep:rustls-pemfile",
@@ -123,14 +123,14 @@
     "hyper-util/http2",
     "hyper-rustls?/http2",
 ]
-http3 = [
-    "rustls-tls-manual-roots",
-    "dep:h3",
-    "dep:h3-quinn",
-    "dep:quinn",
-    "dep:slab",
-    "dep:futures-channel",
-]
+#http3 = [
+#    "rustls-tls-manual-roots",
+#    "dep:h3",
+#    "dep:h3-quinn",
+#    "dep:quinn",
+#    "dep:slab",
+#    "dep:futures-channel",
+#]
 json = ["dep:serde_json"]
 #macos-system-configuration = ["dep:system-configuration"]
 multipart = ["dep:mime_guess"]
@@ -382,9 +382,9 @@
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.h3]
-version = "0.0.6"
-optional = true
-
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.h3-quinn]
-version = "0.0.7"
-optional = true
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.h3]
+#version = "0.0.6"
+#optional = true
+
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.h3-quinn]
+#version = "0.0.7"
+#optional = true
 
@@ -452,14 +452,14 @@
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies.pin-project-lite]
 version = "0.2.11"
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.quinn]
-version = "0.11.1"
-features = [
-    "rustls",
-    "runtime-tokio",
-]
-optional = true
-default-features = false
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.quinn]
+#version = "0.11.1"
+#features = [
+#    "rustls",
+#    "runtime-tokio",
+#]
+#optional = true
+#default-features = false
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies.rustls]
 version = "0.23.4"
@@ -483,9 +483,9 @@
 features = ["alloc"]
 optional = true
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.slab]
-version = "0.4.9"
-optional = true
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.slab]
+#version = "0.4.9"
+#optional = true
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies.tokio]
 version = "1.0"
